src/output/plugins/httpd/HttpdInternal.hxx

```diff
@@ -115,6 +115,12 @@ private:
 	 */
 	char const *const website;
 
+	/**
+	 * The configured password.
+	 */
+	char const *password;
```

Comment:
This field should only exist when the feature is available and it should be `const`